src/app/shared/utils/consts.ts

```diff
@@ -84,22 +84,42 @@ export const Consts = {
                 label: 'S3 GLACIER',
                 value: 'GLACIER'
             }
+        ],
+        'azure-blob' : [
+            {
+                label: 'Archive',
+                value: 'Archive'
+            }
+        ],
+        'gcp-s3' : [
+            {
+                label: 'Archive',
+                value: 'Archive'
+            }
+        ]
+    },
+    RETRIEVAL_OPTIONS: {
+        'aws-s3' : [
+            {
+                label: "Expedited",
+                value: "Expedited"
+            },
+            {
+                label: "Standard",
+                value: "Standard"
+            },
+            {
+                label: "Bulk",
+                value: "Bulk"
+            }
+        ],
+        'azure-blob' : [
```

Comment:
the storage class change to Hot as well Cold too

Reply:
Do you mean add both options here as below:
```
'azure-blob' : [
            {
                label: "Hot",
                value: "Hot"
            },
            {
                label: "Cold",
                value: "Cold"
            }
        ]
```

Reply:
yes